internal/future/serving/client/grpc.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2019 Google LLC
```

Comment:
Merge this into /rpc/ directory and remove all exported methods that are only needed because they are in separate packages.

Reply:
Do I need if it make sense to merge the grpc client and http client into one file?